docs/BUGS.md

```diff
@@ -0,0 +1,272 @@
+                                  _   _ ____  _
+                              ___| | | |  _ \| |
+                             / __| | | | |_) | |
+                            | (__| |_| |  _ <| |___
+                             \___|\___/|_| \_\_____|
+
```

Comment:
I would suggest to either remove the ASCII art or put it into a codeblock by three back ticks.

Reply:
But why? lines prefixed with 4 spaces get shown as "code" already. [see here](https://github.com/curl/curl/blob/08046135919b4334a06b16fc03649e3e50944bcf/docs/BUGS.md)

Reply:
I think I should just get rid of it anyway....

Reply:
> But why? lines prefixed with 4 spaces get shown as "code" already. [see here](https://github.com/curl/curl/blob/08046135919b4334a06b16fc03649e3e50944bcf/docs/BUGS.md)

That's why I also approved. It was a non blocking stylish thing :)